bin/torii/build.rs

```diff
@@ -0,0 +1,30 @@
+use std::env;
+use std::error::Error;
+
+use vergen::{BuildBuilder, Emitter};
+use vergen_gitcl::GitclBuilder;
+
+fn main() -> Result<(), Box<dyn Error>> {
+    let build = BuildBuilder::default().build_timestamp(true).build()?;
+    let gitcl =
+        GitclBuilder::default().describe(true, true, None).branch(true).sha(true).build()?;
+
+    // Emit the instructions
+    Emitter::default().add_instructions(&build)?.add_instructions(&gitcl)?.emit_and_set()?;
+
+    let version = env!("CARGO_PKG_VERSION");
+    let git_branch = env::var("VERGEN_GIT_BRANCH").unwrap_or("unknown".to_string());
+    let git_sha = env::var("VERGEN_GIT_SHA").unwrap_or("unknown".to_string());
+    let git_describe = env::var("VERGEN_GIT_DESCRIBE").unwrap_or("unknown".to_string());
+
+    let dev = git_describe.contains(&git_sha) || git_describe.contains("dirty");
```

Comment:
I've built on main merging this PR + no dirty repo, and I'm still having `dev`.
<img width="332" alt="image" src="https://github.com/user-attachments/assets/bba5affb-3787-40ae-a12d-9372a6e35ed7" />

Maybe we can skip the dev and stick to the branch + commit hash?

Reply:
still this is why it's dev, as soon as you are not on a tag / dirty we consider it as dev. you built it yourself from main so this is why you still get the dev. its mostly because if we always show the branch, if you are on an official release (so a tag), it will show `torii 1.4.0 (1.4.0 hash)` which i find a bit weird. but ig we could remove it